### DISCUSSION TITLE
Couldn't run google colab examples from git hub.pybamm

### DISCUSSION THREAD

**Discussion by reenubosepybamm** [pybamm-team/PyBaMM#2521] [General]
%pip install pybamm -q    # install PyBaMM if it is not installed
import pybamm
import numpy as np
import matplotlib.pyplot as plt
GIVING AN ERROR
PluginNotFound                            Traceback (most recent call last)
[<ipython-input-5-103384696aab>](https://localhost:8080/#) in <module>
      2 get_ipython().run_line_magic('pip', 'install pybamm --no-cache -q')
      3 get_ipython().run_line_magic('pip', 'install pybamm -q    # install PyBaMM if it is not installed')
----> 4 import pybamm
      5 import numpy as np
      6 import matplotlib.pyplot as plt

6 frames
[/usr/local/lib/python3.8/dist-packages/pybtex/plugin/__init__.py](https://localhost:8080/#) in _load_entry_point(group, name, use_aliases)
     79         for entry_point in pkg_resources.iter_entry_points(search_group, name):
     80             return entry_point.load()
---> 81     raise PluginNotFound(group, name)
     82 
     83 

PluginNotFound: plugin pybtex.database.input.bibtex not found

**Comment by brosaplanella** [comment]
This seems to be a bug so I have now opened an issue about it #2524
